### ISSUE TITLE
Confusing menu highlighting

### ISSUE THREAD

**Issue by viasux** [ajbura/cinny#432]
### Describe the problem

The colors on the "Select your role" selection when creating a space are confusing. I would assume the darker option is selected, but the lighter option is the selected one.

### Describe the solution you'd like

![image](https://user-images.githubusercontent.com/47131867/160224891-8bf20064-a22e-4ddb-bcf7-664bcd30c6df.png)
Swap the colors (meaning Admin would be selected in this instance)

**Comment by viasux** [comment]
Also, I'm unclear on why there's a distinction between admin and founder here, maybe an info hover dialogue thing could be inserted next to it, explaining the difference?

**Comment by kfiven** [comment]
About the text, does https://github.com/ajbura/cinny/commit/30f89307731bb6242821ea91e8f00598dae06d5e resolve your concerns?

**Comment by viasux** [comment]
> About the text, does [30f8930](https://github.com/ajbura/cinny/commit/30f89307731bb6242821ea91e8f00598dae06d5e) resolve your concerns?

Yes, however, it should be "Founder (101) overrides the default Admin (100) power level." instead of "Founder (101) override the default Admin (100) power level."

**Comment by greentore** [comment]
I agree this switch looks very confusing.
